RULES.md

```diff
@@ -34,6 +34,7 @@ Rules are declared in the [`Notice` module](https://github.com/MobilityData/gtfs
 | [E030](#E030) | Inconsistent field `agency_timezone` | 
 | [E031](#E031) | Invalid `agency_id` | 
 | [E032](#E032) | `calendar.txt` `end_date` is before `start_date` |
+| [E032](#E033) | `route_id` not found |
 | [E034](#E034) | Invalid `shape_id` |
```

Comment:
We should standardize on the "missing ID" rule titles and descriptions. For example, I believe E033 and E034 are semantically the same for route_id and shape_id respectively, but the title and description of the rules are significantly different. The E034 description is really hard to read, in particular.

Reply:
Definitely yes, I'm currently working on this PR. The changes will be pushed later.